api-spec/api-spec.md

```diff
@@ -0,0 +1,47 @@
+
+In WFS3 Collections are the sets of data that can be queried ([7.11](https://rawgit.com/opengeospatial/WFS_FES/master/docs/17-069.html#_feature_collections_metadata)), and each describes basic information about the geospatial dataset, like its name and description, as well as the spatial and temporal extents of all the data contained. [STAC collections](../collections-spec/README.md) contain this same information, along with other STAC specific fields and thus are compliant with both WFS Collections and STAC Collections and is returnd from the `/collections/{collection_id}` endpoint.
+
+In WFS3 Features are the individual records within a Collection and are provided in GeoJSON format. [STAC Items](../item-spec/README.md) are analagous to WFS3 Features, are in GeoJSON, and are returned from the `/collections/{collection_id}/items/{item_id}` endpoint.
+
+### WFS3 Endpoints
+
+The core WFS3 endpoints are shown below, with details provided in an [OpenAPI specification document](definitions/WFS3.yaml).
+
+| Endpoint      | Returns          | Description        |
+| ------------ | ------------- | ---------------------- |
+| / | JSON        | Landing page, links to API capabilities |
+| /conformance | JSON | Info about standards the API conforms to       |
+| /collections | Collections | List of Collections contained in the catalog |
+| /collections/{collection_id} | Collection | Returns single Collection JSON |
+| /collections/{collection_id}/items | Items | GeoJSON FeatureCollection of Items in Collection |
+| /collections/{collection_id}/items/{item_id} | Item | Returns single Item GeoJSON |
+
+The `/collections/{collection_id}/items` endpoint accepts parameters for filtering the results (also called filters). Searching using POST will accept a JSON object where the top level keys specify which type of filter to apply (e.g., bbox=[...]). Those same key names can be used as query string parameters for the GET request. 
+
+Items in the collection should match all filters to be returned when querying. This implies a logical AND operation. If an OR operation is needed, it should be specified through an extension filter.
+
+### Filter Parameters
+
+| Parameter      | Type          | Description        |
+| ------------ | ------------- | ---------------------- |
+| bbox | [number] (x4)        | Requested bounding box [west, south, east, north] |
+| time | datetime string | Single date, date+time, or a range ('/' seperator) |
```

Comment:
In the rest of the API we just use string as data type and define the string type (datetime) with links to the corresponding RFC in the description. Would just copy that over from the other specs.

Reply:
+1